sql_generators/experiment_monitoring/templates/templating.yaml

```diff
@@ -9,10 +9,10 @@ queries:
     per_app: false
   experiment_events_live_v1:
     per_app: true
-    start_date: '2023-03-13'
+    start_date: "2023-03-13"
```

Comment:
```suggestion
    start_date: "2023-10-10"
```

Reply:
The reason for the start date here is to save some cost. BigQuery will run a backfill automatically back to the start date, I don't think it's a big issue here.

Reply:
We probably want to backfill at least back to when the Cirrus experiment started, right? So `2023-10-02`. This is also not specific to Cirrus so I didn't think to change the date at all.

Reply:
The dashboards only query the two most recent days from the `live` tables, the rest is coming from the `aggregates` tables (which we'd need to be backfilling)

Reply:
Oh makes sense, thanks!